nasa-gibs/worldview
# Wv 2375.build overhaul

## Description

Fixes WV-2375 

## How To Test

- Build the application, check for any build errors 
- With application built, spot check the application for differences to production

## Bugs found / to fix
- [x] Axios sometimes errors out with 404
- [X] E2E Chrome test fails on moveTo, moveToElement, getAttributes
This was a nightwatch config issue
- [x] E2E FireFox random test failures 
- [x] Fix layers/layers-sidebar-test in mac/chrome
- [x] Fix measure/measure-test in mac/chrome
